components/Home/Home.vue

```diff
@@ -1,5 +1,24 @@
 <template>
   <div class="fixed w-full h-full overflow-hidden flex flex-col">
+    <t-full-modal ref="detailModal" :hide-close-button="true">
```

Comment:
This change has still nothing to to here.

Reply:
this modal comes from the branch dev2

Reply:
I just switched the PR to target dev2.